modules/@apostrophecms/attachment/index.js

```diff
@@ -122,6 +123,10 @@ module.exports = {
       'migrate-from-disabled-file-key': {
         usage: 'Usage: node app @apostrophecms/attachment:migrate-from-disabled-file-key\n\nThis task should be run after removing the disabledFileKey option from uploadfs.\nIt should only be relevant for storage backends where\n' + 'that option is not mandatory, i.e. only local storage as of this writing.',
         task: self.migrateFromDisabledFileKeyTask
+      },
+      'recompute-all-doc-references': {
+        usage: 'Recompute mapping between attachments and docs,\nshould only be needed for rare repair situations',
```

Comment:
Why would this need to stay around as a task?

Reply:
People have been known to manipulate mongo directly and could miss a nuance like the attachment reference tracking.